<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>918</width>
    <height>532</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTextEdit" name="l_dbName">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>162</y>
      <width>271</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="list">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>31</y>
      <width>271</width>
      <height>411</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>121</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Enter database name</string>
    </property>
   </widget>
   <widget class="QLabel" name="label3">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>271</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Enter eployee name</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="l_employee">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>312</y>
      <width>271</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>381</y>
      <width>261</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Get</string>
    </property>
   </widget>
   <widget class="QLabel" name="label1">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>40</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Enter server name</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="l_servName">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>81</y>
      <width>271</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>918</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
